lab/experiments/ContentAnimateIn.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+	<div :class="$s.ProductLoad">
+		<div
+			:class="$s.ProductGrid"
+		>
+			<div
+				v-for="index in 16"
+				:key="index"
+				:class="$s.Product"
+			>
+				<template
+					v-if="!reveal"
+				>
+					<m-skeleton-block :class="$s.ProductImage" />
+					<m-skeleton-text />
+				</template>
+				<m-transition-staggered
+					:stagger-item-count="staggerItemCount"
```

Comment:
I'm always looking for ways that we can simplify public api and make it easier to understand. My primary concern at the start was this complex prop and understanding how to use it. When testing locally and removing this prop, I didn't noticed a difference on either mobile or desktop with how the staggered transition loaded.

If this wasn't a configuration, what are we missing out on?

Reply:
This property tells it how many items per row are being staggered in the animation. If it's not provided, it just animates all the items in at the same time without staggering.

with prop
![staggered](https://user-images.githubusercontent.com/1486885/144526661-4828fdf1-dca5-4d62-98ae-2f227708afc8.gif)

without prop
![unstaggered](https://user-images.githubusercontent.com/1486885/144526683-97eb0d60-452e-4c19-8755-1bb63ed485fd.gif)